tests-manual/debugger/README.md

```diff
@@ -0,0 +1,45 @@
+Debugger Command Line Test Programs
+===================================
+
+This directory contains a set of utilities used to manually test the pipeline debugger features. These tests should be run against a live system with the debugger facilities built into it.
+
+To start testing;
+
+  - Change directory to this directory
+
+  - Type the command
+
+    .. code-block:: console
+
+       ./debug <ServiceName>
+
+    Where *<ServiceName>* is the name of a south or north service. You will need to quote the name if it contains whitespace or wildcard characters that have meaning to the shell.
+
+   - Attach the debugger to the pipeline by usign the attach command
+
+    .. code-block:: console
+
+       attach
+
+The debugger is now attached to the pipeline and collecting one reading at each point in the pipeline.
+
+To see the list of command available type the commands command
+
+.. code-block:: console
+
+    % commands
+    attach:		Attach the pipeline debugger
+    buffer:		Return the contents of the buffers at every pipeline element
+    detach:		Detch the debugger from the pipeline
+    isolate:	Isolate the pipeline from the destination
+    replay:		Replay the buffered data though the pipeline
+    resumeIngest:	Resume the flow of data into the pipeline
+    setBuffer:	Set the number of reading to hold in each buffer, passed an integer argument
+    state:		Return the state of the debugger
+    step:		Allow readings to flow into the pipeline. Passed an optonal number of readings to ingest, default to 1 if omitted
```

Comment:
Passed an optonal number of readings to ingest, default to 1 if omitted
Change to:
Pass an optional number of readings to ingest; default to 1 if omitted

Reply:
Still there